x-pack/plugins/observability/public/pages/slos/components/slo_list_item.tsx

```diff
@@ -82,11 +83,14 @@ export function SloListItem({ slo, onDeleted, onDeleting }: SloListItemProps) {
                   <EuiLink onClick={handleNavigate}>{slo.name}</EuiLink>
                 </EuiFlexItem>
 
-                <EuiFlexItem grow={false}>
-                  <div>
-                    <SloBadges slo={slo} />
-                  </div>
-                </EuiFlexItem>
+                <EuiFlexGroup direction="row" responsive={false} gutterSize="m">
```

Comment:
Interesting; why not include `SloForecastedBadge` inside `SloStatusBadges` and keep this the same?

Reply:
Ah, I see now.

I would propose we keep a wrapping component for all the badges that will be rendered (so a status badge but also these ones:

<img width="264" alt="Screenshot 2022-12-27 at 17 03 25" src="https://user-images.githubusercontent.com/535564/209691421-21e232ac-c86a-4c2e-9a19-4e8a2f5ebdc3.png">

Reply:
Moved to `SloBadges` 👍🏻